tests/e2e/scripts/fast_test.sh

```diff
@@ -36,14 +36,22 @@ then
     --image-url=nginx \
     --kube-master="https://$MASTER_IP:6443" \
     --kubeconfig=$KUBECONFIG \
-    --test.v \
-    2>&1 | tee -a /tmp/testcase.log
```

Comment:
Is it possible to still store the log in this file for debugging?

Reply:
can use `> /tmp/testcase.log` to replace `tee -a `, but i think this is duplicated. the log will be stored in action logs directly, we can look at action logs.

Reply:
but if we use `>`, i'm not quite sure whether the logs will be printed on action std out

Reply:
by test, if we use `>`, ginkgo will not print logs to the action stdout
cannot think a proper way to store the log in the file and use the ginkgo exit code meanwhile, so just logs to the action stdout directly

Reply:
If users run e2e in local host, the log file should be still needed, we can fix the e2e bug first, then explore a way to store the log file.

Reply:
Open an issue to track it #4174 @gy95